inference-engine/tests/functional/plugin/shared/include/behavior/config.hpp

```diff
@@ -73,8 +72,12 @@ namespace BehaviorTestsDefinitions {
         SKIP_IF_CURRENT_TEST_IS_DISABLED()
         // Create CNNNetwork from ngrpah::Function
         InferenceEngine::CNNNetwork cnnNet(function);
-        ASSERT_THROW(auto execNet = ie->LoadNetwork(cnnNet, targetDevice, configuration),
-                     InferenceEngine::Exception);
+        if (targetDevice.find(CommonTestUtils::DEVICE_AUTO) != std::string::npos) {
```

Comment:
Why we skip a test for the AUTO, it should throw just like any other plugin

Reply:
This test is modified in a early commit which has no exception, but I add the exception for unsupported config as CVS-57233. I created ticket #CVS-57552 to track this.